e2etest/test_manager.go

```diff
@@ -82,13 +85,19 @@ func initBTCClientWithSubscriber(t *testing.T, cfg *config.Config) *btcclient.Cl
 // StartManager creates a test manager
 // NOTE: uses btc client with zmq
 func StartManager(t *testing.T, numMatureOutputsInWallet uint32, epochInterval uint) *TestManager {
-	btcHandler := NewBitcoindHandler(t)
-	btcHandler.Start()
+	manager, err := container.NewManager()
+	require.NoError(t, err)
+
+	btcHandler := NewBitcoindHandler(t, manager)
+	bitcoind := btcHandler.Start(t)
 	passphrase := "pass"
 	_ = btcHandler.CreateWallet("default", passphrase)
 
 	cfg := defaultVigilanteConfig()
 
+	cfg.BTC.Endpoint = fmt.Sprintf("127.0.0.1:%s", bitcoind.GetPort("18443/tcp"))
+	cfg.BTC.ZmqSeqEndpoint = fmt.Sprintf("tcp:// 127.0.0.1:%s", bitcoind.GetPort("28333/tcp"))
```

Comment:
do we still support zmq?

Reply:
Yes, depending [on config](https://github.com/babylonlabs-io/vigilante/blob/49e8cb8297df6b3cb411ac1224c4c7588f85e264/btcclient/notifier.go#L118)